---a/net/minecraft/block/state/pattern/BlockHelper.java
+++b/net/minecraft/block/state/pattern/BlockHelper.java
@@ -1,31 +1,27 @@
 package net.minecraft.block.state.pattern;
 
-import com.google.common.base.Predicate;
 import net.minecraft.block.Block;
 import net.minecraft.block.state.IBlockState;
 
-public class BlockHelper implements Predicate
-{
+import com.google.common.base.Predicate;
+
+public class BlockHelper implements Predicate {
+
     private final Block block;
     private static final String __OBFID = "CL_00002020";
 
-    private BlockHelper(Block blockType)
-    {
+    private BlockHelper(Block blockType) {
         this.block = blockType;
     }
 
-    public static BlockHelper forBlock(Block blockType)
-    {
+    public static BlockHelper forBlock(Block blockType) {
         return new BlockHelper(blockType);
     }
 
-    public boolean isBlockEqualTo(IBlockState state)
-    {
+    public boolean isBlockEqualTo(IBlockState state) {
         return state != null && state.getBlock() == this.block;
     }
 
-    public boolean apply(Object p_apply_1_)
-    {
-        return this.isBlockEqualTo((IBlockState)p_apply_1_);
+    public boolean apply(Object p_apply_1_) {
+        return this.isBlockEqualTo((IBlockState) p_apply_1_);
     }
-}
+}
